sdv/multi_table/base.py

```diff
@@ -142,12 +143,16 @@ def validate(self, data):
                 self._table_synthesizers[table_name].validate(table_data)
 
             except (InvalidDataError, ValueError) as error:
-                error_msg = str(error)
-                error_msg = error_msg.replace(
-                    'The provided data does not match the metadata:',
-                    f"Table: '{table_name}'"
-                )
+                if isinstance(error, InvalidDataError):
+                    error_msg = f"Table: '{table_name}'"
+                    for _error in error.errors:
+                        error_msg += f'\nError: {_error}'
+
+                else:
+                    error_msg = str(error)
```

Comment:
Instead of using an if statement, can we just have separate catch statements for the different error type? ie.
```python
except InvalidDataError as error:
    error_msg = f"Table: '{table_name}'"
    for _error in error.errors:
        error_msg += f'\nError: {_error}'
except ValueError as error:
    error_msg = str(error)